de/manual/ch_Business.docbook

```diff
@@ -1,1279 +1,1976 @@
-        <para>The prebuilt Business Account hierarchy will not meet your needs exactly. You will need to make
-          adjustments for the hierarchy to function well with your particular situation. It should
-          be close enough that it is recommended you begin with it.
-        </para>
-      </note>
-<!--
-      <para>To use &app;&rsquo;s integrated accounts
-      payable system, you must first setup an account of the <emphasis>A/Payable</emphasis>
-      type. The <emphasis>A/Payable</emphasis> account is usually a sub-account
-      under Liabilities. It is within this account that the integrated A/P system
-      places transactions.
-      <screen>Basic A/P Account Hierarchy:
-
--Assets
-  -Bank
--Liabilities
-  -Accounts Payable
--Expenses
-  -AP Expenses</screen>
-</para>
-
-     <para>You need to add additional asset accounts and real expense accounts to make
-     this hierarchy useful.  The important aspects of this hierarchy are that you need
-     an expense account and an Accounts Payable account, with account type set to
-     <emphasis>A/Payable</emphasis>.</para>
--->
-    </sect2>
-
-    <sect2 id="busnss-ar-setuptaxtables">
-      <title>Tax Tables</title>
-
-      <para>Tax Tables can used to determine the tax for customer invoices (or vendor bills).
-      </para>
-
-      <para>A tax table can be assigned to an invoice line or bill line.
-      </para>
-
-      <para>The default invoice line tax table can be assigned to each customer and the default bill line tax
-        table can be assigned to each vendor.
-      </para>
-
-      <para>The default tax table for new customers or new vendors can be specified in the <emphasis>Book
-        Options</emphasis> window which can be accessed by &mc.fi.new;
-        <guilabel>Business</guilabel> tab.
-      </para>
-
-      <para>Tax Tables are maintained using the <emphasis>Sales Tax Table</emphasis> editor which is accessed
-        via menu &mc.bus.tax-tab; . The following fields should be entered:
-      </para>
+    <para>Primärer Zweck des Geschäftsmoduls ist es, den Überblick bei seinen unbezahlten Rechnungen —
+      von Lieferanten und an Kunden — zu behalten. Die Verwaltung der Lieferantenrechnungen kann
+      auch für Privatanwender interessant sein.
+    </para>
 
+    <para>Dabei erleichtert die Verwaltung zugehöriger Stammdaten die Vorgänge. Dazu gehören folgende
+      Tabellen:
       <itemizedlist>
         <listitem>
-          <para><guilabel>Name</guilabel> This is the tax table name.
-          </para>
-        </listitem>
-
-        <listitem>
-          <para><guilabel>Type</guilabel> Either <guilabel>Percent %</guilabel> or <guilabel>Value $</guilabel>.
-          </para>
-        </listitem>
-
-        <listitem>
-          <para><guilabel>Value</guilabel>This is the percentage or value depending on <guilabel>Type</guilabel>.
+          <para>Verkaufssteuern — MWSt. in der EU —, welche die Erfassung auf separaten Konten ermöglicht, und
```

Comment:
Nach meinem Verständnis werden Spiegelstriche um Einschübe im 

- Deutschen wie Klammern, also ohne Leerzeichen innen `Anfang —Einschub— Ende`, aber im
- Englischen wie Komma, also Leerzeichen nur danach `Anfang— Einschub— Ende`,
gesetzt. M.E. kann man dann im Englischen gleich Komma  nehmen.

Reply:
Hier stehen Beispiele im Duden: [https://www.duden.de/sprachwissen/rechtschreibregeln/gedankenstrich](https://www.duden.de/sprachwissen/rechtschreibregeln/gedankenstrich)